lib/model/query/entities.js

```diff
@@ -111,6 +129,12 @@ const createMany = (dataset, rawEntities, sourceId, userAgentIn) => async ({ all
   const creatorId = context.auth.actor.map((actor) => actor.id).orNull();
   const userAgent = blankStringToNull(userAgentIn);
 
+  // Validate UUID with the purged entities
+  const purgedUuids = await _getPurgedUuids(all, rawEntities.map(e => e.uuid));
+  if (purgedUuids.length > 0) {
+    throw Problem.user.uniquenessViolation({ fields: 'uuid', values: purgedUuids.join(', ') });
```

Comment:
I'm not sure what this should say given that the existing message mentions a single entity uuid. 

"The entity with UUID (uuid1, uuid2, uuid3,...) has been deleted."

But it might be better than what it currently returns, e.g. this response json: 

```
{'code': 409.3,
 'details': {'fields': 'uuid',
             'values': '88ed7209-cb54-4b7e-98ed-e1cc26a93cc6, '
                       'd8b37943-9202-44ce-837b-8be85a34d373'},
 'message': 'A resource already exists with uuid value(s) of '
            '88ed7209-cb54-4b7e-98ed-e1cc26a93cc6, '
            'd8b37943-9202-44ce-837b-8be85a34d373.'}
```

Reply:
updated with `entityUniquenessViolationWithDeleted`